docs/step-by-step/kubeapps-on-tkg/README.md

```diff
@@ -0,0 +1,72 @@
+- You have access to the [VMWare Cloud Services Portal (CSP)](https://console.cloud.vmware.com/). If not, talk to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html) to request access.
+- You have access to, at least, the Tanzu™ Application Catalog™ Demo environment. If not, reach out to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html).
+- You have the _kubectl_ CLI and the Helm v3.x package manager installed. Learn how to [install kubectl and Helm v3.x](https://docs.bitnami.com/kubernetes/get-started-kubernetes/#step-3-install-kubectl-command-line).
+
+## Intended Audience
+
+This information is intended for administrators who want to install Kubeapps on a Tanzu Kubernetes Grid cluster and use it to deploy and manage applications from the VMware Marketplace™ and the VMware Tanzu™ Application Catalog™. Besides, this content is also intended for application administrators and developers who want to use Kubeaps to deploy and manage modern apps in a Kubernetes architecture.
+
+The information is written for users who have a basic understanding of Kubernetes and are familiar with container deployment concepts. In-depth knowledge of Kubernetes is not required.
+
+# Steps
+
+<!-- To be extracted to a separate file -->
+
+## Step 1: Prepare the identity management
+
+<!-- Work in progress -->
+
+### Install or use the existing Pinniped in TKG?
+
+- [Enabling Identity Management in Tanzu Kubernetes Grid](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.3/vmware-tanzu-kubernetes-grid-13/GUID-mgmt-clusters-enabling-id-mgmt.html).
+- [Tanzu Kubernetes Grid 1.3 with Identity Management](https://liveandletlearn.net/post/kubeapps-on-tanzu-kubernetes-grid-13/)
+
+### Configure an OIDC provider
+
+- [VMware Cloud Services as OIDC provider](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md#vmware-cloud-services)
+
+<!-- To be extracted to a separate file -->
+
+## Step 2: Configure and install Kubeapps
+
+<!-- Work in progress -->
+
+- [Configure Pinniped to trust the ODIC provider](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider-with-pinniped.md#configure-pinniped-to-trust-your-oidc-identity-provider)
+- [Configuring Kubeapps to proxy requests via Pinniped](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider-with-pinniped.md#configuring-kubeapps-to-proxy-requests-via-pinniped)
+- Use the VMware custom look&feel <!-- Undocumented -->
+
+<!-- To be extracted to a separate file -->
+
+## Step 3: Add the VMware Marketplace™ as an application repository
+
+<!-- Work in progress -->
+
+- [Adding an public application repository](https://github.com/kubeapps/kubeapps/blob/master/docs/user/dashboard.md)
+
+<!-- To be extracted to a separate file -->
```

Comment:
Guessing we'll want to include deploying an app from each application repository after adding it?

Reply:
I don't think this is necessary, we can point to https://github.com/kubeapps/kubeapps/blob/master/docs/user/dashboard.md which seems to cover this in detail. Or at most, we can reuse/interpolate that content here rather than creating it separately.

If we decide we would like to include this, it would be Step 5: Deploy an application on TKG using Kubeapps